server/models/procedures/trial_balance.sql

```diff
@@ -130,6 +130,22 @@ BEGIN
     JOIN period p ON pj.period_id = p.id
     WHERE p.locked = 1 GROUP BY pj.record_uuid;
 
+  -- check that there are no transactions with accounts of locked creditor groups
+  INSERT INTO stage_trial_balance_errors
+    SELECT pj.record_uuid, MAX(pj.trans_id), 'POSTING_JOURNAL.ERRORS.LOCKED_CREDITOR_GROUP_ACCOUNT' AS code
+    FROM posting_journal AS pj JOIN stage_trial_balance_transaction AS temp
```

Comment:
I just tested it locally and it works very well, but I would like to know why the temporary table stage_trial_balance_transaction was created